.run/crowpi-debug.run.xml

```diff
@@ -1,5 +1,5 @@
 <component name="ProjectRunConfigurationManager">
-  <configuration default="false" name="crowpi-examples [debug]" type="MavenRunConfiguration" factoryName="Maven">
+  <configuration default="false" name="CrowPi Debug" type="MavenRunConfiguration" factoryName="Maven">
```

Comment:
Do you know that if you change this name you need to update screenshots and documentation as well?

Reply:
I'm well aware of that and noticed that due to now skipping tests by default and a couple other changes I have to touch these parts of the documentation anyways, so I will quickly update the screenshots too. This is already a part of the updated documentation PR I'm working on.